getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -99,8 +99,10 @@ You should see a wide grey slab that covers the grid and blue and red axes in
 the viewport.
 
 We're going to move the ground down so we can see the floor grid. Select the
-``Ground`` node, hold the :kbd:`Ctrl` key down to turn on grid snapping,
-and click and drag down on the Y axis. It's the green arrow in the move gizmo.
+``Ground`` node, click on the *Use Snap* button (or press the :kbd:`Y` key) to turn on grid snapping,
+and click and drag down on the Y-axis. It's the green arrow in the move gizmo.
```

Comment:
Ctrl still works for grid snapping so we shouldn't completely remove references to it.
```suggestion
``Ground`` node, holding the :kbd:`Ctrl` key down turns on grid snapping while it's held. You can also
click on the *Use Snap* button (or press the :kbd:`Y` key) to turn on grid snapping.
With snapping enabled using either method, click and drag down on the Y-axis. It's the green arrow in the move gizmo.
```

Reply:
Took this into account and rewrote the section.